saleor/webhook/payloads.py

```diff
@@ -301,43 +309,55 @@ def generate_product_deleted_payload(product: "Product", variants_id):
 
 
 PRODUCT_VARIANT_FIELDS = (
-    "name",
     "sku",
+    "name",
+    "track_inventory",
     "private_metadata",
     "metadata",
 )
 
 
-def generate_product_variant_payload(product_variant: "ProductVariant"):
+def generate_product_variant_listings_payload(variant_channel_listings):
```

Comment:
Did you test this?

Reply:
Yep

https://github.com/mirumee/saleor/pull/7330/files#diff-a7128589cc3915d1eee316b65f943f6ef9a4e366fed1434a9db0798b55ee72feR121